### ISSUE TITLE
Multiple Accounts

### ISSUE THREAD

**Issue by Gretzky17** [AHAAAAAAA/PokemonGo-Map#2853]
Does the 2nd/3rd/etc accounts have to be set to different locations?  Can I just have them scanning from the same origin at varying times/speeds to pick up all the pokemon faster?

**Comment by failuretolaunch** [comment]
Yes, you could do that. They don't have to be different locations.
